spec/facets/ExtractionErrorRunFacet.json

```diff
@@ -0,0 +1,51 @@
+{
+	"$schema": "https://json-schema.org/draft/2020-12/schema",
+	"$id": "https://openlineage.io/spec/facets/1-0-0/ExtractionErrorRunFacet.json",
+	"$defs": {
+		"ExtractionErrorRunFacet": {
+			"allOf": [{
+				"$ref": "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/RunFacet"
+			}, {
+				"type": "object",
+				"properties": {
+					"totalTasks": {
+						"description": "The number of distinguishable tasks that were processed by OpenLineage, whether successfully or not. Those could be, for example, distinct SQL statements.",
+						"type": "integer"
+					},
+					"failedTasks": {
+						"description": "The number of distinguishable tasks that were processed not successfully by OpenLineage. Those could be, for example, distinct SQL statements.",
+						"type": "integer"
+					},
+                    "errors": {
+                        "type": "array",
+                        "items": {
+                            "type" : "object",
+                            "properties" : {
+                                "errorMessage" : {
+                                    "description" : "Text representation of extraction error message.",
+                                    "type" : "string"
+                                },
+                                "task": {
```

Comment:
How would that look exactly? Let's say parsing SQL fails - should this be a manually-named name of the stage, e.g. `SQL Parser`?

Reply:
No, this should be a SQL statement that failed.